packages/tabs/src/useTabs.ts

```diff
@@ -59,6 +59,7 @@ export function useTabs<Item = any>({
   const getTabListProps = ({ role = 'tablist', ...other }: React.HTMLProps<any> = {}) => {
     return {
       role,
+      'aria-orientation': 'horizontal',
```

Comment:
Since `aria-orientation` is removed from the `useSelection` hook, a default `aria-orientation` is returned from this prop getter. Consumers have the option to override this value if they choose to.

Reply:
Could this use the `vertical` prop to apply `vertical | horizontal` based on its state? This would make it so consumers don't have to worry about it.